From b450eb85f2938eaff4ea7c0eaad1706d38959549 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 2 Jun 2017 16:27:14 +0800
Subject: [PATCH 1907/5242] MLK-15001-3 arm64: fsl-imx8qm.dtsi: Remove lvds
 nodes

commit  f9466a1485efe0c42cb07045cd3d8286fdf9ecf3 from
https://source.codeaurora.org/external/imx/linux-imx.git

No driver works with the lvds nodes, so let's remove them.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    8 ------
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi      |   30 --------------------
 2 files changed, 38 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 7e49216..f9008a10 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -310,14 +310,6 @@
 	status = "okay";
 };
 
-&lvds0 {
-	status = "okay";
-};
-
-&lvds1 {
-	status = "okay";
-};
-
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index f1a8d5c..ed92223 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1019,36 +1019,6 @@
 		status = "disabled";
 	};
 
-	lvds0: lvds@56241000 {
-		compatible = "fsl,imx8qm-lvds";
-		reg = <0x0 0x56241000 0x0 0x1000>;
-		interrupts = <0 57 4>;
-		clocks =
-			<&clk IMX8QM_LVDS0_PIXEL_CLK>,
-			<&clk IMX8QM_LVDS0_PHY_CLK>;
-		clock-names = "clk_pixel", "clk_phy";
-		power-domains = <&pd_lvds0>;
-		instance = <0>;
-		data-width = <24>;
-		data-mapping = "jeida";
-		status = "disabled";
-	};
-
-	lvds1: lvds@57241000 {
-		compatible = "fsl,imx8qm-lvds";
-		reg = <0x0 0x57241000 0x0 0x1000>;
-		interrupts = <0 58 4>;
-		clocks =
-			<&clk IMX8QM_LVDS1_PIXEL_CLK>,
-			<&clk IMX8QM_LVDS1_PHY_CLK>;
-		clock-names = "clk_pixel", "clk_phy";
-		power-domains = <&pd_lvds1>;
-		instance = <1>;
-		data-width = <24>;
-		data-mapping = "jeida";
-		status = "disabled";
-	};
-
 	mipi0: mipi@56220000 {
 		compatible = "fsl,imx8qm-mipi_dsi";
 		reg = <0x0 0x56220000 0x0 0x10000>;
-- 
1.7.9.5

